src/main/java/com/dev/cinema/controllers/UserController.java

```diff
@@ -22,9 +23,9 @@ public UserController(UserService userService, UserMapper userMapper) {
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto get(@RequestParam String email) {
-        return userService.findByEmail(email)
-                .map(userMapper::toUserResponseDto)
-                .orElseThrow();
+    public UserResponseDto get(Authentication authentication) {
```

Comment:
let's not change this - this method should return info about any user by his email